src/main/java/org/kiwiproject/test/junit/jupiter/MongoServerExtension.java

```diff
@@ -115,7 +125,28 @@ public MongoServerExtension() {
      * @param dropTime when to drop the test database
      */
     public MongoServerExtension(DropTime dropTime) {
+        this(dropTime, DEFAULT_SERVER_VERSION);
+    }
+
+    /**
+     * Creates a new instance that will drop and re-create the test database after each test with the given {@link ServerVersion}.
+     *
+     * @param serverVersion the version of the mongo server to use.
+     */
+    public MongoServerExtension(ServerVersion serverVersion) {
```

Comment:
Presumably a service using this extension would be using the same version of Mongo for all data access. If there are multiple DAOs in the service/application, and assuming a test per DAO, then each DAO test would (assuming they want to use a different version than the default) need to specify that same version. Would it be worth adding a way to specify the Mongo version globally for all those tests, e.g. `-DmongoServerExtension.mongoVersion=MONGO_3_6`? If this makes any sense, we can add it as a new enhancement issue.

Reply:
We can talk about adding this, but not in this PR